client/src/tests/Login.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import Login from "./../pages/Login";
+import { Input } from "@material-ui/core";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+it("Login Page renders correctly", () => {
+  const tree = renderer.create(<Login />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+it("Login Page redirects given valid user", async () => {
+  const validUser = "Admin";
+  const mountedWrapper = Enzyme.mount(<Login />);
+  const input = mountedWrapper.find("#username").last();
```

Comment:
Try to find by type/class here - it should be do-able

Reply:
Since we are targeting a specific element should we select by Id rather than Class? For context, there are multiple elements rendered with the same set of classes.

Reply:
yeah then id is fine!